RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -192,11 +71,67 @@
-                                                                           'keep recoPhotonCores_islandPhotonCore_*_*',
-                                                                           'keep recoPhotons_islandPhotons_*_*'
-                                                                           ])
-                    )
+    e.toModify( RecoEgammaAOD.outputCommands, 
+                func=lambda outputCommands: outputCommands.extend(['keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppGED_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerpp_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppIsland_*_*',
+                                                                   'keep recoPhotonCores_islandPhotonCore_*_*',
+                                                                   'keep recoPhotons_islandPhotons_*_*'])
+               )
+# RECO content
+RecoEgammaRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gedPhotonCore_*_*',
+        'keep *_gedPhotons_*_*',
+        #'keep *_gedPhotonsTmp_*_*',        
+        'keep recoPhotons_mustachePhotons_*_*',
+        'keep recoPhotonCores_mustachePhotonCore_*_*',
+        #'drop *_gedPhotonsTmp_valMapPFEgammaCandToPhoton_*',
+        'keep recoTrackExtras_ckfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_ckfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfInOutTracksFromConversions_*_*',
+        'keep recoConversions_uncleanedOnlyAllConversions_*_*',
+        'keep recoTrackExtras_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_uncleanedOnlyCkfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfInOutTracksFromConversions_*_*',
+        'keep recoRecoEcalCandidates_hfRecoEcalCandidate_*_*')                                                                 
+)
+RecoEgammaRECO.outputCommands.extend(RecoEgammaAOD.outputCommands)
+
+# Full Event content 
+RecoEgammaFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gsfElectronCores_*_*', 
+        'keep *_gsfElectrons_*_*', 
+        'keep *_uncleanedOnlyGsfElectronCores_*_*', 
+        'keep *_uncleanedOnlyGsfElectrons_*_*', 
+        'keep *_eidRobustLoose_*_*',
+        'keep *_eidRobustTight_*_*',
+        'keep *_eidRobustHighEnergy_*_*',
+        'keep *_eidLoose_*_*',
+        'keep *_eidTight_*_*',
+        'keep *_egmGedGsfElectronPF*Isolation_*_*',
+        'keep *_egmGsfElectronIDs_*_*', 
+        'keep *_egmPhotonIDs_*_*',
+        'keep *_conversions_*_*',
```

Comment:
The line
```
        'drop *_conversions_uncleanedConversions_*',
```
applied the ```drop``` in the AOD event content before this ```keep``` here for FEVT: it should be repeated here, then

Reply:
By the way, I would personally move L31 right below current L28, so that it is clear that the ```drop``` actually applies on the line right above it.

Reply:
In AOD and RECO event contents, there is only no specific label name as following `vector<reco::Conversion>              "conversions"               ""                "RECO"`
So we can remove `'drop *_conversions_uncleanedConversions_*',` command in AOD PSet and remove the wildcard in L28  'keep recoConversions_conversions__*',


vector<reco::Conversion>              "conversions"               "uncleanedConversions"   "RECO"

Reply:
I miss how do you get rid of ```*_conversions_uncleanedConversions_*``` here in the FEVT event content, without the ```drop``` statement...

Reply:
Sorry.. My mistake. I thought there was no drop in FEVT. Then, I will keep "drop" and change this file as you suggested earlier.

Reply:
Plese re-drop ```*_conversions_uncleanedConversions_*``` here, since this ```keep``` follows the ```drop``` in the AOD section